opencodeblocks/blocks/block.py

```diff
@@ -25,6 +25,16 @@
 
 BACKGROUND_COLOR = QColor("#E3212121")
 
+DEFAULT_BLOCK_DATA = {
+    "title": "_",
+    "splitter_pos": [88, 41],
+    "width": 618,
+    "height": 184,
+    "metadata": {"title_metadata": {"color": "white", "font": "Ubuntu", "size": 10}},
+    "sockets": [],
+}
+NONE_OPTIONAL_FIELDS = {"block_type", "position"}
```

Comment:
Do not write unnecessary constants, here defaults are already defined in the Block class, you should use those and not duplicate values.

Reply:
Shouldn't this be in OCBBlock DEFAULT_DATA following the Serializable refactor ?